Gemfile

```diff
@@ -75,7 +75,7 @@ end
 
 gem "phlex-rails"
 
-gem "rbui", github: "rbui-labs/rbui", branch: "main"
-# gem "rbui", path: "../rbui"
+gem "ruby_ui", github: "rbui-labs/ruby_ui", branch: "main"
+# gem "ruby_ui", path: "../ruby_ui"
```

Comment:
there is still a lot of typo that rbui should be ruby_ui, but it deserves another PR